UnityProject/Assets/Scripts/Construction/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -42,6 +43,13 @@ private void OnDisable()
 		if(isServer) UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
 	}
 
+	// Only runs in editor - useful for updating the sprite direction
+	// when the initial direction is altered via inspector.
+	private void OnValidate()
+	{
+		RefreshSprites();
```

Comment:
maybe have it a editor only thing? 
with the 
#if unity_editor

Reply:
Hah, ReiderJack suggested to use OnValidate instead of the if editor region thing

Reply:
Just saying for unforeseen consequences in runtime
but If it doesn't cause anything like that it should be okay